OctaneVSPlugin/OctaneServices.cs

```diff
@@ -329,7 +329,9 @@ public void RemoveFromMyWork(UserItem entity)
         public async Task RemoveCommentFromMyWork(BaseEntity entity)
         {
             RestConnector.AwaitContinueOnCapturedContext = false;
-            await es.DismissCommentByIdAsync(workspaceContext, entity.Id);
+            string putUrl = workspaceContext.GetPath() + "/comments/" + entity.Id + "/dismiss";
```

Comment:
can be a separate method somewhere

Reply:
response is ignored, is it like this for every dismiss, maybe we can show a notification that it was removed